zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -6184,6 +6183,12 @@ components:
         - FAILED
         - UNSPECIFIED
         - UNKNOWN
+    SortOrderEnum:
+      description: The Sort Order.
+      enum:
+        - asc
+        - desc
+      default: asc
```

🔧 I believe that going upper case here would be more common, right?
```suggestion
        - ASC
        - DESC
      default: ASC
```

---

I followed the requirements over the issue https://github.com/camunda/camunda/issues/20512 
I can move it all to ASC and DESC with no problems